app/assets/stylesheets/modules/filters.css.scss

```diff
@@ -68,6 +68,7 @@
   max-height: 20vh;
   max-width: $filters-width - 2 * $margin-default;
   overflow: auto;
+  z-index: 8;
```

Comment:
why 8? 
I'd prefer to use a variable that would show the rule

Reply:
I agree :) Would you be willing to take care of that? The reason for that 8 was that without it the autocomplete dropdown is hidden behind the filters.